tests/functional/aws-node-sdk/test/support/awsConfig.js

```diff
@@ -0,0 +1,20 @@
+const AWS = require('aws-sdk');
```

Comment:
this file seems to be a duplicate of the function in tests/functional/aws-node-sdk/test/support/config.js.  I would remove that function in that file and import your function into that file so we don't duplicate it.